lib/player.js

```diff
@@ -2125,6 +2146,14 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
     track.mode = 'hidden';
     this.eventManager_.listen(track, 'cuechange', () => {
       for (const cue of track.activeCues) {
+        const eventName = shaka.Player.EventName.Metadata;
```

Comment:
BTW, is this whole method only applicable to Safari or is it gonna work for all the browsers?

Reply:
Right now only Safari, but it will work in all browsers, mux.js exposes the ID3 metadata but it is still not handled in the app.

Reply:
Gotcha. Is it related to [this PR](https://github.com/google/shaka-player/pull/2501) or is that separate?

Reply:
That PR is the base, but you still have to process the data and expose it to the SS and launch the metadata event. I had planned to work on this soon, but I have two more urgent PR for this week according to our needs.